bin/brew

```diff
@@ -21,6 +21,7 @@ symlink_target_directory() {
 BREW_FILE_DIRECTORY="$(quiet_cd "${0%/*}/" && pwd -P)"
 HOMEBREW_BREW_FILE="${BREW_FILE_DIRECTORY%/}/${0##*/}"
 HOMEBREW_PREFIX="${HOMEBREW_BREW_FILE%/*/*}"
+HOMEBREW_TEMP="${HOMEBREW_TEMP:-/private/tmp}"
```

Comment:
Will this upset Linux? Worth setting in `brew.sh` to a different default for macOS and not-macOS?

Reply:
Yeah, `/private/tmp` is macOS only. Moving it to `brew.sh` sounds good.